packages/blue-sdk/src/addresses.ts

```diff
@@ -492,11 +481,70 @@ entries(permissionedWrapperTokens).forEach(([chainId, tokens]) => {
  */
 export const convexWrapperTokens: Record<number, Set<Address>> = {
   [ChainId.EthMainnet]: new Set([
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDTWBTCWETH-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDCWBTCWETH-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvCRVUSDTBTCWSTETH-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxTryLSD-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDETHCRV-morpho"],
-    addressesRegistry[ChainId.EthMainnet]["stkcvx2BTC-f-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDTWBTCWETH-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDCWBTCWETH-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvCRVUSDTBTCWSTETH-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxTryLSD-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvxcrvUSDETHCRV-morpho"],
+    _addressesRegistry[ChainId.EthMainnet]["stkcvx2BTC-f-morpho"],
   ]),
 };
+
+export let addressesRegistry = deepFreeze(_addressesRegistry);
+export let addresses = addressesRegistry as Record<number, ChainAddresses>;
+export let unwrappedTokensMapping = deepFreeze(_unwrappedTokensMapping);
+
+/**
+ * Registers custom addresses and unwrapped token mappings to extend
+ * the default address registry (on ewisting or unknown chains).
+ *
+ * @param options - Optional configuration object
+ * @param options.unwrappedTokens - A mapping of chain IDs to token address maps,
+ *                                  where each entry maps wrapped tokens to their unwrapped equivalents.
+ * @param options.customAddresses - Custom address entries to merge into the default registry.
+ *                                  Can be a subset of `ChainAddresses` if chain is already known.
+ *                                  Must provide all required addresses if chain is unknown.
+ *
+ * @throws {Error} If attempting to override an existing address.
+ *
+ * @example
+ * ```ts
+ * registerCustomAddresses({
+ *   customAddresses: {
+ *     1: { contract: "0xabc..." }
+ *   },
+ *   unwrappedTokens: {
+ *     1: { "0xWrapped": "0xUnwrapped" }
+ *   }
+ * });
+ * ```
+ */
+export function registerCustomAddresses({
+  unwrappedTokens,
+  customAddresses,
+}: {
+  unwrappedTokens?: Record<number, Record<Address, Address>>;
+  customAddresses?:
```

Comment:
```suggestion
  addresses?:
```

Just to have the same naming of consumer and override 🥺